_docs/concepts/network-and-auth/auth.md

```diff
@@ -71,7 +71,7 @@ Istio Auth provides a per-cluster CA (Certificate Authority) to automate key and
 
 
 
-*   Generate a key and certificate pair for each service account
+*   Generate a [SPIFFE](spiffe.github.io/docs/svid) key and certificate pair for each service account
```

Comment:
Please change it to `https://spiffe.github.io/docs/svid` to stay consistent with the rest of this file.